docs/cmd/kn_source_cronjob.md

```diff
@@ -0,0 +1,34 @@
+## kn source cronjob
+
+CronJob source command group
+
+### Synopsis
+
+CronJob source command group
+
+```
+kn source cronjob [flags]
+```
+
+### Options
+
+```
+  -h, --help   help for cronjob
+```
+
+### Options inherited from parent commands
+
+```
+      --config string       kn config file (default is $HOME/.kn/config.yaml)
+      --kubeconfig string   kubectl config file (default is $HOME/.kube/config)
+      --log-http            log http traffic
+```
+
+### SEE ALSO
+
+* [kn source](kn_source.md)	 - Event Source command group
+* [kn source cronjob create](kn_source_cronjob_create.md)	 - Create a Crontab scheduler source.
```

Comment:
Some use `Crontab scheduler source` and some use `CronJob source`. I think it's better to use a consistent word.

Reply:
> Just as knative serving API invocations go to `pkg/serving`, I think all eventing server side invocation methods shall go to `pkg/eventing`. Because some day, we may want to expose these server side invocations as a client-go library. Forgive me if I'm wrong. I think it's one of team decisions.

I might have missed that discussion, do you have some pointers for me ? 

The real design decision is whether we want to make the sources support self-contained and all under one package (which make it easier to refactor e.g. when they will be moved away from eventing proper), or whether we want expose them as API and maintaining it (in that case we should build up the same structure as for serving).

I personally tend to the 'self-contained' approach for sources, but would also be happy to go over eventing pkg.

Reply:
But yeah, probably better to be consistent. Let me change this later. thanks